src/server/common/helpers/proxy/proxy-fetch.js

```diff
@@ -0,0 +1,27 @@
+import { config } from '~/src/config/config.js'
```

Comment:
given undici is setup with the proxy globally and the forward-proxy will route internal urls direct, could we remove proxy-fetch all together?

Reply:
Happy to. I just copied the changes that you'd applied to portal-frontend. If its not needed i'll remove this file

Reply:
At the moment, if a tenant uses `import { fetch } from 'undici'` then because of the `setGlobalDispatcher(new ProxyAgent(proxyUrl))` calls will use the `ProxyAgent`  Dispatcher

If a tenant doesn't use `import { fetch } from 'undici'` and the http lib they use doesn't internally hook up to the `setGlobalDispatcher(new ProxyAgent(proxyUrl))` then calls will not use the `ProxyAgent`  Dispatcher

I wonder if its worth:
- Adding some code that does a fetch
- Writing a wrapper or example of how to provide a ProxyAgent dispatcher to a http lib?

Reply:
Sounds good, but think i'll need your help with that.

Reply:
> Sounds good, but think i'll need your help with that.

Moved this to [the main thread](https://github.com/DEFRA/cdp-node-frontend-template/pull/133#issuecomment-2618720427)